tests/legacy-cli/e2e/tests/experimental/ivy-lazy-load.ts

```diff
@@ -5,23 +5,88 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import { readdirSync } from 'fs';
-import { prependToFile, replaceInFile } from '../../utils/fs';
+import { prependToFile, replaceInFile, writeFile } from '../../utils/fs';
 import { ng } from '../../utils/process';
-import { createProject } from '../../utils/project';
+import { createProject, updateJsonFile } from '../../utils/project';
+import { expectToFail } from '../../utils/utils';
 
-export default async function() {
-  await createProject('ivy-project', '--enable-ivy');
+export async function ivyLazyLoadSetup(projectName: string) {
+  // Make Ivy project.
+  await createProject(projectName, '--enable-ivy');
```

Comment:
Shouldn’t we create a routing application by default? 

Doing so will allow us to remove certain logic from this file

Reply:
Done